Traceback (most recent call last):
  File "training_script.py", line 7, in <module>
    import matplotlib.pyplot as plt
ImportError: No module named matplotlib.pyplot
Traceback (most recent call last):
  File "transfer_script.py", line 7, in <module>
    import matplotlib.pyplot as plt
ImportError: No module named matplotlib.pyplot
Traceback (most recent call last):
  File "transfer_unfreezeall_script.py", line 7, in <module>
    import matplotlib.pyplot as plt
ImportError: No module named matplotlib.pyplot
